models/migrations/v36.go

```diff
@@ -5,11 +5,13 @@
 package migrations
 
 import (
-	"code.gitea.io/gitea/models"
+	"context"
+
+	repo_module "code.gitea.io/gitea/modules/repository"
 
 	"xorm.io/xorm"
 )
 
 func regenerateGitHooks36(x *xorm.Engine) (err error) {
-	return models.SyncRepositoryHooks()
+	return repo_module.SyncRepositoryHooks(context.Background())
```

Comment:
Probably should use the Shutdown context here.

Reply:
done.